autogpt_platform/frontend/src/components/agptui/Button.tsx

```diff
@@ -20,6 +20,9 @@ const buttonVariants = cva(
         ghost:
           "hover:bg-neutral-100 text-[#272727] dark:text-neutral-100 dark:hover:bg-neutral-700",
         link: "text-[#272727] underline-offset-4 hover:underline dark:text-neutral-100",
+        library_outline:
```

Comment:
is 'outline' used anywhere and if so why don't these match?

Reply:
yeah we shouldn't have separate classes of components for *part* of the application